package.json

```diff
@@ -1,29 +1,79 @@
 {
-    "name": "type-detect",
-    "version": "1.0.0",
-    "description": "Improved typeof detection for node.js and the browser.",
-    "author": "Jake Luer <jake@alogicalparadox.com> (http://alogicalparadox.com)",
-    "license": "MIT",
-    "keywords": [],
-    "repository": {
-        "type": "git",
-        "url": "git@github.com:chaijs/type-detect.git"
-    },
-    "engines": {
-        "node": "*"
-    },
-    "main": "./index",
-    "scripts": {
-        "test": "make test"
-    },
-    "dependencies": {},
-    "devDependencies": {
-        "component": "*",
-        "coveralls": "2.11.8",
-        "jscoverage": "0.3.7",
-        "mocha": "*",
-        "mocha-lcov-reporter": "0.0.1",
-        "mocha-phantomjs": "*",
-        "simple-assert": "*"
+  "name": "type-detect",
+  "version": "1.0.0",
```

Comment:
I feel like `2.0.0` would be the correct version number here, since this PR has breaking changes.